parse_lib.py

```diff
@@ -106,6 +106,12 @@ def update_anchor_href(anchor):
         anchor['target'] = '_blank'
 
 
+def convert_to_img(html, equation):
```

Comment:
Definitely can improve the name-- `convert_obj_to_img` might be more appropriate, and I can just change the `equations` variable to `object_tags` to be more generalized. I stuck with `equations` originally because equations were the only things contained in `object` tags, but that's a brittle relationship that could change later.